docs/quickstart/image_signing.md

```diff
@@ -0,0 +1,138 @@
+# Image signing with cosign
+
+This document includes the following:
+* Guide to generate a cosign-signed image.
+* Guide to config image security policy to enable signature verification of the image.
+* Guide to distribute the public key and image security policy via offline-fs-kbs.
+
+## Signing the encrypted/unencrypted image and enable signature verification when running the workload
+
+When an image is from a protected registry, the image needs to be applied some defined security strategies.
```

Comment:
Just linking Steve's review for this section for readability.
https://github.com/confidential-containers/documentation/pull/65/files#r1025358425

Reply:
Thanks, have fixed it